mongodb/Module.md

```diff
@@ -1,37 +1,27 @@
-# Ballerina MongoDB Module
-
-Connects to MongoDB from ballerina 
-
-## Module Overview
-
+## Overview
 The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
 
-## Prerequisites
+This module supports Ballerina Swan Lake Beta2 version.
```

Comment:
Since we are mentioning this as a prerequisite, I don't think we need to mention it here as well.

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.